lunavl/sdk/indexes/builder.py

```diff
@@ -0,0 +1,112 @@
+"""
+Module contains core index builder.
+"""
+from pathlib import Path
+from typing import Union
+
+from FaceEngine import PyIFaceEngine
+
+from lunavl.sdk.descriptors.descriptors import (
+    FaceDescriptor,
+    FaceDescriptorBatch,
+)
+from lunavl.sdk.errors.errors import LunaVLError
+from lunavl.sdk.errors.exceptions import LunaSDKException
+from .base import CoreIndex
+from .stored_index import DynamicIndex, IndexType, DenseIndex
+
+
+class IndexBuilder(CoreIndex):
+    """
+    Index builder class (only supports face descriptors)
+
+    Attributes:
+        _bufSize (int): storage size with descriptors
+    """
+
+    def __init__(self, faceEngine: PyIFaceEngine):
+        super().__init__(faceEngine.createIndexBuilder(), faceEngine)
+        self._bufSize = 0
+
+    @property
+    def bufSize(self) -> int:
```

Comment:
for what?

Reply:
to get information about the size of index (other indexes have this information)